.gitignore

```diff
@@ -1,6 +1,7 @@
 # The build output should clearly not be checked in
+/bazel-*
 /python/ray/core
-/python/ray/pyarrow_files/pyarrow/
```

Comment:
Not sure if this it's intentional, but currently all the pyarrow files go directly under `/python/ray/pyarrow_files/` and not `/python/ray/pyarrow_files/pyarrow`